src/pages/Calendar/Caption/caption.style.ts

```diff
@@ -0,0 +1,82 @@
+import styled from "styled-components/native";
+import { ColorScheme } from "@styles/globalStyles";
+import { FontScheme } from "@styles/globalStyles";
+
+export const Sc = {
+  Captions: styled.View`
+    flex-direction: row;
+    gap: 96px;
+  `,
+
+  LeftCaptions: styled.View`
+    flex-direction: column;
+  `,
+
+  RightCaptions: styled.View`
+    flex-direction: column;
+  `,
+
+  GroupCaptions: styled.View`
+    flex-direction: row;
+    margin-bottom: 13px;
+    gap: 12px;
+  `,
+
+  Container: styled.View`
+    width: 358px;
+    height: 112px;
+    margin: 0 auto;
```

Comment:
This one shouldn't be necessary. Setting fixed widths and heights will end up causing issues down the line. Also, since you are already using `justify-content: center` and `align-items: center`, setting inline margins to auto would essentialy do nothing